dotcom-rendering/src/web/components/TopicFilterBank.tsx

```diff
@@ -52,14 +57,17 @@ export const TopicFilterBank = ({ availableTopics, format }: Props) => {
 			</div>
 
 			<div css={topicStyles}>
-				{availableTopics.slice(0, 5).map((topic) => {
+				{availableTopics.slice(0, 5).map((topic: Topic) => {
+					const buttonParams = `${topic.type}:${topic.value}`;
+					const isActive = selectedTopics?.includes(buttonParams);
```

Comment:
Appreciate this is mentioned in the PR but I think this is going to be a problem once we add in key events as we'll need the two button types to be in sync with each other.

Reply:
I think each button click will set new params: this would override whatever the previous params were so everything would always be in sync